bcda/cli.go

```diff
@@ -466,3 +482,29 @@ func deleteDirectoryContents(dirToDelete string) (filesDeleted int, err error) {
 
 	return len(files), nil
 }
+
+func cleanupCCLF(cclfmap map[string][]*cclfFileMetadata) {
+	for _, cclflist := range cclfmap {
+		for _, cclf := range cclflist {
+			newpath := fmt.Sprintf("%s/%s", os.Getenv("PENDING_DELETION_DIR"), cclf.name)
+			if !cclf.imported {
+				// check the timestamp on the failed files
+				elapsed := time.Since(cclf.deliveryDate).Hours()
+				if int(elapsed) > (24 - deleteThresholdHr) {
+					err := os.Rename(cclf.filePath, newpath)
```

Comment:
I think we should log this.  This is an indication that the file is doomed (we retried it for a 24 hour period, and still it won't ingest - so we need to specify in logs that we are deleting a file that was never ingested).

Reply:
good catch!